tests/func/bad/bad_func.catala_en

```diff
@@ -27,21 +27,16 @@ $ catala test-scope R
 [RESULT] r = 30
 ```
 
-Note: ideally this could use test-scope but some positions are lost during translation to lcalc
-
 ```catala-test-inline
-$ catala interpret -s S
-[ERROR] There is a conflict between multiple valid consequences for assigning
-  the same variable.
+$ catala test-scope S
+[ERROR] Error during evaluation: two or more concurring valid computations.
```

Comment:
This error message should be more pedagogical ; why not leave the exact same text as the old message? It seems like it would work.